pkg/plugins/policies/meshloadbalancingstrategy/api/v1alpha1/meshloadbalancingstrategy.go

```diff
@@ -32,6 +32,7 @@ type Conf struct {
 	// LocalityAwareness contains configuration for locality aware load balancing.
 	LocalityAwareness *LocalityAwareness `json:"localityAwareness,omitempty"`
 	// LoadBalancer allows to specify load balancing algorithm.
+	// +kuma:non-mergeable-struct
```

Comment:
do we need both `// +kuma:non-mergeable-struct` and `// +kuma:discriminator`?

Reply:
I don't see 

```
// +kuma:discriminator
```

there so maybe it was removed before you commented

Reply:
The `// +kuma:discriminator` is on the `LoadBalancer.Type` https://github.com/kumahq/kuma/pull/12844/files#diff-86958ebfece1357edccc524d537c4b81080e7e946a81cdfa99e2ac74d683bd6fR126

Reply:
oh yeah, you're completely right